src/main/java/uk/gov/hmcts/reform/cwrdapi/controllers/CaseWorkerRefUsersController.java

```diff
@@ -259,4 +272,59 @@
         return ResponseEntity.status(resource.getStatusCode()).body(resource);
     }
 
+    @ApiOperation(
+            value = "This API returns the Staff(Case Worker) profiles based on Service Name and Pagination parameters",
+            authorizations = {
+                    @Authorization(value = "ServiceAuthorization"),
+                    @Authorization(value = "Authorization")
+            }
+    )
+    @ApiResponses({
+            @ApiResponse(
+                    code = 200,
+                    message = "The Staff profiles have been retrieved successfully",
+                    response = StaffProfileWithServiceName.class
+            ),
+            @ApiResponse(
+                    code = 400,
+                    message = BAD_REQUEST
+            ),
+            @ApiResponse(
+                    code = 401,
+                    message = UNAUTHORIZED_ERROR
+            ),
+            @ApiResponse(
+                    code = 403,
+                    message = FORBIDDEN_ERROR
+            ),
+            @ApiResponse(
+                    code = 404,
+                    message = NO_DATA_FOUND
+            ),
+            @ApiResponse(
+                    code = 500,
+                    message = INTERNAL_SERVER_ERROR
+            )
+    })
+    @GetMapping(
+            path = "/get-users-by-service-name",
```

Comment:
i think should be path = "/users" as per Rest standards. Design also we need to put this one.

Reply:
I think it is already under /users
Full url - `/refdata/case-worker/users//get-users-by-service-name`
so you want it to be something like this
`/refdata/case-worker/users/fetchByServiceName`

Reply:
could be like  "/refdata/case-worker/usersByService/"

Reply:
we have decided to go with `/refdata/case-worker/users/staffByServiceName`